hipify_torch/hipify_python.py

```diff
@@ -767,6 +783,13 @@ def c2_repl(m):
                 return CAFFE2_MAP[m.group(0)]
             output_source = RE_CAFFE2_PREPROCESSOR.sub(c2_repl, output_source)
 
+    ## replace all custom mappings.
```

Comment:
Let's move this section to apply the custom mappings *before* any of the default mappings.

Reply:
Done.